demo/vl-rich-data-table.html

```diff
@@ -871,7 +871,7 @@ <h2>Project details</h2>
                                     <input is="vl-input-field" id="filterOpNaamManager" type="text" name="manager.lastName" value="" block/>
                                 </div>
                             </section>
-                            <div>
+                            <div data-vl-hidden-in-modal>
```

Comment:
Ik had alleen het attribuut rechtstreeks op de button in gedachten, maar dit geeft de gebruiker nog meer vrijheid, dus prima.